features/events/syslog/pom.xml

```diff
@@ -260,5 +266,30 @@
         </exclusion>
       </exclusions>
     </dependency>
+    
+    <!-- HZN-742 -->
+    <dependency>
+	 <groupId>org.apache.curator</groupId>
+	 <artifactId>curator-test</artifactId>
+	 <version>2.0.0-incubating</version>
+	 <scope>test</scope>
+	</dependency>
+    
+    <dependency>
+	 <groupId>log4j</groupId>
+	 <artifactId>log4j</artifactId>
+	 <version>1.2.15</version>
```

Comment:
Is there a reason to define the version explicitly?
We usually either use the version-properties e.g. ${log4jVersion}(see https://github.com/OpenNMS/opennms/blob/develop/pom.xml#L1282) defined in the root pom, or do not define a version at all - as it is defined in the root pom in most cases.